cli/command/cli.go

```diff
@@ -27,26 +27,28 @@ import (
 
 // Streams is an interface which exposes the standard input and output streams
 type Streams interface {
-	In() *InStream
-	Out() *OutStream
+	In() InputStreamer
+	Out() OutputStreamer
 	Err() io.Writer
 }
 
 // Cli represents the docker command line client.
 type Cli interface {
 	Client() client.APIClient
-	Out() *OutStream
+	Out() OutputStreamer
```

Comment:
I think you can avoid this change by adding this to `cli/command/in_test.go`

```go

func NewStubInStream(isTerminal bool) *InStream {
    return &InStream{
        isTerminal: isTerminal,
        in: ioutil.NopCloser(strings.NewReader("")),
    }
}
```

and using that stream in the `FakeCli`.

You can probably leave the existing `NewFakeCli()` as is, and add a new `NewFakeCliWithStreams()` that takes all the streams, not just stdout.